doc/indexing.rst

```diff
@@ -395,6 +395,22 @@ These methods may also be applied to ``Dataset`` objects
     ds = da.to_dataset(name="bar")
     ds.isel(x=xr.DataArray([0, 1, 2], dims=["points"]))
 
+Vectorized indexing may be used to extract information from the nearest
+grid cells of interest, for example, the nearest climate model grid
+cells to a collection specified weather station latitudes and longitudes.
```

Comment:
It seems like that section should be merged with "Vectorized Indexing" which could then be renamed to "Pointwise (or vectorized) Indexing)"?

Reply:
agreed. Also, the term `vectorized indexing` is not really explained (there's a comment), so it might be good to explain that somewhere. This sounds like a lot of work, though, so it might be better to do that in a different PR.

Reply:
Interesting discussion. At this point, I don't think I have much to add here. So I concur that this should be addressed in a different PR.